src/OrchardCore/OrchardCore.Abstractions/Json/Serialization/DateTimeJsonConverter.cs

```diff
@@ -16,9 +16,15 @@ public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, Jso
             throw new ArgumentException("Unexpected type to convert.", nameof(typeToConvert));
         }
 
-        if (!reader.TryGetDateTime(out var value) && DateTime.TryParse(reader.GetString()!, out value))
+        if (!reader.TryGetDateTime(out var value))
         {
-            return value;
+            var stringValue = reader.GetString(); 
+            if (DateTime.TryParse(stringValue, out value))
+            {
+                return value;
+            }
+
+            throw new JsonException($"Unable to convert \"{stringValue}\" to DateTime.");
```

Comment:
This can just throw the exception without a message. See https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/converters-how-to?pivots=dotnet-8-0#jsonexception

Reply:
Looks like , the default message does not include the original value ，
If the error message contains the original value, it can help us locate the problem quickly. Especially when working with large amounts of data

```
The unsupported member type is located on type
'System.Collections.Generic.Dictionary`2[Samples.SummaryWords,System.Int32]'.
Path: $.TemperatureRanges | LineNumber: 4 | BytePositionInLine: 24
```